CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.6.0] - 2017-04-04
+### Changed
```

Comment:
regarding bumping up the version and changelog etc, there are a few other places to look at, see https://github.com/altmetric/identifiers/commit/53ceb69d4de655ee86fe4bf4b7e4ad953b5e1029